test/cases/base_tests.sh

```diff
@@ -27,7 +27,7 @@ run_test_case () {
     echo "🏃🏻 Running test: ${TEST_NAME}"
     test_divider
 
-    if sudo "${1}" -test.v | tee "${WORKSPACE}"/"${TEST_NAME}".log; then
```

Comment:
Two suggestions:
 - drop $WORKSPACE entirely (also in other places, if it is used, but that could be independent :) )
 - use a proper temporary directory that is created with a random name and cleaned up, otherwise we risk tests writing stuff on top of each other

Reply:
Makes sense, but this will make logs collection more complicated. See the updated patch.